dashboard/src/App.js

```diff
@@ -48,20 +58,31 @@ const App = () => {
     <div className="App">
       <BrowserRouter>
         <Routes>
-          <Route path="/" element={<MainLayout />}>
-            <Route index element={<TableWithFavorite />} />
-            <Route element={<ProtectedRoute />}>
-              <Route
-                path={APP_ROUTES.USER_PROFILE}
-                element={<ProfileComponent />}
-              />
-              <Route path="dashboard" element={<OverviewComponent />} />
+          <Route path="/" element={<HomeRoute />}></Route>
+          <Route path="/dashboard">
```

Comment:
Should this be using `{APP_ROUTES.HOME}` for consistency instead of the literal `/dashboard`?

But I like the nested routing model here...

Reply:
Done